### PR TITLE
Day one Kotlin for Hacktoberfest

### PR DESCRIPTION
**Pull request by d-l-mcbride**
# Please make sure you have done the following:

- [x] Created a folder with the name of language you code inside directory for the day you are making the contribution to
- [x] Added the source code file inside `/Day<n>/<Language>/`
- [x] Tested your code
- [x] Added credits (author and date) at the top of the code
- [x] Updated README.md for `/Day <n>/`
- [x] Added yourself as a contributor using `all-contributors-cli`

### CODE REVIEW

**Comment by d-l-mcbride** [comment]
Here is my new pull request. Sorry about the last one without pulling from this repo first :(

File: Day1/Kotlin/FizzBuzz.kts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            it % 15 == 0 -> "FizzBuzz"
+            it % 5 == 0 -> "Buzz"
+            it % 3 == 0 -> "Fizz"
+            else -> "${it}"
+        }
+        println(output)
+    }
+}
+
+fizzBuzz((1..15).toList())
```
Comment (aaditkamat) [inline Day1/Kotlin/FizzBuzz.kts:21]: If I'm not wrong, a top level declaration is required.

```suggestion
fun main() {
       fizzBuzz((1..15).toList())
}
```

File: Day1/README.md [hunk 2]
```diff
@@ -595,6 +595,36 @@ int main()
...
+            it % 15 == 0 -> "FizzBuzz"
+            it % 5 == 0 -> "Buzz"
+            it % 3 == 0 -> "Fizz"
+            else -> "${it}"
+        }
+        println(output)
+    }
+}
+
+fizzBuzz((1..15).toList())
```
Comment (aaditkamat) [inline Day1/README.md:625]: Should reflect the change made in the source code file

**Review by aaditkamat — CHANGES_REQUESTED** [review]
I've suggested some minor changes. Apart from those, I don't think any other amendments are required since the logic of the program is correct.

File: Day1/Kotlin/FizzBuzz.kts (hunk 1 above)
Comment (d-l-mcbride) [reply Day1/Kotlin/FizzBuzz.kts:21]: For Kotlin script file(.kts) stand alone functions work without the associated main function. I can make the change though.

Comment (aaditkamat) [reply Day1/Kotlin/FizzBuzz.kts:21]: In that case you can leave it as it is.

**Review by aaditkamat — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**